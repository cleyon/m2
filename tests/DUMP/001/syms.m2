@define foo 1
@define bar 2
@longdef baz
Line A
Line B
@endlong
Symbols:
@dump symbols
End Symbols
