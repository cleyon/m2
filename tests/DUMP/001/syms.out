@define bar	2
@longdef baz
Line A
Line B
@endlongdef
@define foo	1
